### ISSUE TITLE
Figure out how to handle Hijri dates before year 1

### ISSUE THREAD

**Issue by cbeer** [sul-dlss/dlme#804]
The Hijiri calendar started at 622 AD (Gregorian), so it makes no sense to display any Hijiri year as a negative number. Any year that started before the Hijiri year "1" is not called a Hijiri year, it is referred to in a different way.

Part of https://github.com/sul-dlss/dlme/issues/800.

**Comment by jacobthill** [comment]
I had a chance to speak with the curatorial advisors and they would like some more time to think through this but also have confirmed that there isn't much of a problem with the way we have approached this. For background the Hijra is the date when Muhammad migrated from Mecca to Medina. This is the beginning of the Islamic calendar, much like our calendar is based on the birth of Christ (even if this is inaccurate). People in the Arab world sometimes need to refer to historical events that happened before the Hijra and it is common enough to say, for example, "200 years before the Hijra." The problem is it has not been formalized the same way CE and BCE have. It was said, however, that its not clear what negative numbers mean in the Gregorian or Hijri calendar view and we need to be more explicit. I have opened a thicket https://github.com/sul-dlss/dlme/issues/829. So the plan for now should be to try to make this more explicit and give the curatorial team a couple months to think about it and try to come up with more improvements for the Spring work cycle.
